releases/patch-releases.md

```diff
@@ -39,6 +39,34 @@ of the actual release.
 Cherry-pick PRs which miss merge criteria will be carried over and tracked
 for the next patch release.
 
+## Support Period
+
+In accordance with the [yearly support KEP][yearly-support], the Kubernetes
+Community will support active patch release series for a period of roughly
+fourteen (14) months.
+
+The first twelve months of this timeframe will be considered the standard
+period.
+
+Towards the end of the twelve month, the following will happen:
+
+- [Release Managers][release-managers] will cut a release
+- The patch release series will enter maintenance mode
+
+During the two-month maintenance mode period, Release Managers may cut
+additional maintenance releases to resolve:
+
+- CVEs (under the advisement of the Product Security Committee)
+- dependency issues (including base image updates)
+- critical core component issues
```

Comment:
Are we going to allow other changes in the maintenance mode period?

I guess that one option is that we might allow them if we are fixing one of those three issues specified here, but otherwise, we wouldn't cut a release only because of other changes. Another option is that we don't allow other changes non-related to those at all.

Should we make that clear in the document?